UnitsNet.NanoFramework/GeneratedCode/Temperature/UnitsNet.NanoFramework.Temperature.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>UnitsNet.nanoFramework.Temperature</id>
+    <version>4.89.0</version>
+    <title>Units.NET - Temperature</title>
+    <authors>Andreas Gullberg Larsen</authors>
+    <owners>Andreas Gullberg Larsen</owners>
+    <license type="expression">MIT</license>
+    <projectUrl>https://github.com/angularsen/UnitsNet</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Temperature unit for .NET nanoFramework. It is light-weight and thoroughly tested.&#10; 
+For C#/VB Universal Windows code (UWP), use UnitsNet instead. Get all the common units of measurement and the conversions between them. </description>
```

Comment:
Description: `Adds temperature units for Units.NET on nanoFramework.`

Reply:
Changed. I left the mention the full package as I believe it's important in case someone tries to use this incidentally.
But it can be remove too, if you prefer.

Reply:
The Universal Windows part is unique to Windows Runtime Component nuget.
How about `Adds temperature units for Units.NET on .NET nanoFramework.&#10; For .NET or .NET Core, use UnitsNet instead.`

Reply:
LGTM! Done.